app/models/piece.rb

```diff
@@ -16,8 +16,15 @@ def is_obstructed?(x,y)
 	end
 
 	def move_to(x,y)
-		update_attributes(x_position: x, y_position: y)
+      updated_status = update_attributes(x_position: x, y_position: y)
```

Comment:
you could probably do it in one line update_attributes(x_position: x, y_position: y, moved: true)